app.js

```diff
@@ -129,6 +70,8 @@ function getEligiblePoolData() {
                     'query': queryMore
                 })
 
+                // console.log(lastId)
```

Comment:
Suggest to delete all console.logs, especially those commented out; unless it helps with debugging.